src/questions/derivatives.yaml

```diff
@@ -0,0 +1,948 @@
+      key: D
+- uuid: 82424bfc-264d-49e7-a0f7-26c9e78b516a
+  type: MultipleChoice
+  content: An asset in a derivative can represent?
+  hint: NoHint
+  explanation: An asset can represent a commodity, a bond, a stock, a cryptocurrency and many more.
+  answerKeys:
+    - A
+    - B
+    - D
+  subTopics:
+    - "introduction-to-derivatives"
+  difficultyLevel: "Low"
+  choices:
+    - content: Commodity
+      key: A
+    - content: Bond
+      key: B
+    - content: IOU Document
+      key: C
+    - content: Stock
+      key: D
+- uuid: 84970ae5-4563-4404-9d8b-b59068386f25
+  type: MultipleChoice
+  content: When a bank manager hedges his portfolio by selling futures, he
+  hint: Hedging is used to reduce the risk of return
+  explanation: When someone hedges their portfolio by selling futures they remove up the opportunity for gains and the probability of loss.
+  answerKeys:
+    - A
+    - B
+  subTopics:
+    - "introduction-to-derivatives"
+  difficultyLevel: "Medium"
+  choices:
+    - content: losses his chances of gains
+      key: A
+    - content: removes his loss chances
+      key: B
+    - content: increases his chances of gains
+      key: C
+    - content: All of the above
+      key: D
+- uuid: 3531d44a-67df-4956-98b3-72b99c02daad
+  type: SingleChoice
+  content: What are the advantages to hedge a portfolio?
+  hint: NoHint
+  explanation: Hedging is done to reduce the risk of a portfolio.
+  answerKeys:
+    - B
+  subTopics:
+    - "introduction-to-derivatives"
+  difficultyLevel: "Low"
+  choices:
+    - content: Maximise the gain chances
+      key: A
+    - content: Minimise the risk chances
+      key: B
+    - content: Profit from capital gains
+      key: C
+    - content: All of the above
+      key: D
+- uuid: e43c62c2-5cf4-4e77-bbb3-8901d01a1263
+  type: SingleChoice
+  content: Which of the following contracts wants the investor to buy assets in future time period?
+  hint: NoHint
+  explanation: TO buy assets in future date the investor must use Long contracts.
+  answerKeys:
+    - C
+  subTopics:
+    - "introduction-to-derivatives"
+  difficultyLevel: "Low"
+  choices:
+    - content: Short
+      key: A
+    - content: Hedge
+      key: B
+    - content: Long
+      key: C
+    - content: Cross
+      key: D
+- uuid: df6a9359-acee-47a6-b0ec-c0969da19a41
+  type: SingleChoice
+  content: In a short contracts the investors
+  hint: NoHint
+  explanation: explanation
+  answerKeys:
+    - A
+  subTopics:
+    - "introduction-to-derivatives"
+  difficultyLevel: "Low"
+  choices:
+    - content: Sell securities in future time
+      key: A
+    - content: Buy securities in future time
+      key: B
+    - content: Future Hedge
+      key: C
+    - content: None of the above
+      key: D
+- uuid: f3e168ef-a8b3-432c-807b-135fd306335d
+  type: SingleChoice
+  content: What are Speculations in Derivative Market?
+  hint: NoHint
+  explanation: Speculations means using market fluctuations to gain profits in long term.
```

Comment:
Speculator form some beliefs and based on that they take a long or short position. Here Option `A` and `B` are partially correct.

We should update explanation and options for this question

Reply:
Done the updation